src/file.cr

```diff
@@ -664,18 +664,23 @@ class File < IO::FileDescriptor
 
   # Write the given *content* to *filename*.
   #
-  # An existing file will be overwritten, else a file will be created.
+  # The *mode* parameter can be used to change the file's `open` mode, e.g. to `"a"` for appending.
```

Comment:
It's doesn't have to be in this PR but there shouldn't be just one example but a list of all possible modes or at least a reference to such a list.

Reply:
Adding it to `open` probably makes the most sense, since `write` is just a thin layer over the former. I think I'll do that in a future PR, though.